queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name like '%mon';
+SELECT name FROM animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth <= '2019-01-01';
```

Comment:
- [ ] Kindly make sure the end date is 2019-31-12 as the question is to list the name of all animals born between 2016 and 2019 so it includes the entire 2019 year, please kindly fix that 😁 😁